tox.ini

```diff
@@ -26,10 +30,10 @@ deps =
     coverage==4.0.1
     mock==1.3.0
     Twisted==19.10.0
-    lint: flake8==3.7.7
-    py37-lint: flake8-isort==2.6.0
-    py37-lint: flake8-commas==2.0.0
-    py37-lint: flake8-bugbear==18.8.0
+    lint: flake8==3.8.3
+    lint-py38: flake8-isort==4.0.0
+    lint-py38: flake8-bugbear==20.1.4
+    lint-py27: flake8-commas==2.0.0
```

Comment:
Should these by pyXX-lint?  Why is the last one still using py27?

Reply:
Python 2.7 allows trailing commas in fewer places than recent Python 3s, so running flake8-commas on a recent Python version might require commas where they are syntactically invalid in Python 2.7.